src/components/backend-ai-session-launcher.ts

```diff
@@ -2746,6 +2794,104 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
     this.shadowRoot.querySelector('#HPCOptimizationOptions').style.display = isOpenMPChecked ? 'none' : 'block';
   }
 
+  /**
+   * Toggle startup code input section according to session type
+   *
+   * @param {Event} e
+   */
+  _toggleStartUpCommandEditor(e) {
+    this.sessionType = e.target.value;
+    const isBatchmode: boolean = (this.sessionType === 'batch');
+    const startUpCommandEditor = this.shadowRoot.querySelector('#batch-mode-config-section');
+    startUpCommandEditor.style.display = isBatchmode ? 'inline-flex' : 'none';
+    if (isBatchmode) {
+      const editor = this.shadowRoot.querySelector('#command-editor');
+      editor.refresh();
+      editor.focus();
+    }
+  }
+
+  /**
+   * Toggle scheduling time when session type is in batch
+   *
+   */
+  _toggleScheduleTime() {
+    this.useScheduledTime = this.shadowRoot.querySelector('#use-scheduled-time').selected;
+    this.shadowRoot.querySelector('vaadin-date-time-picker').style.display = this.useScheduledTime ? 'block': 'none';
+    this._getSchedulableTime();
```

Comment:
Do we need to call `_getSchedulableTime` when `useScheduledTime` is off?

Reply:
Nope, we don't. I just fixed this issue using `setInterval` and `clearInterval` function.